Draft-Accepted/RFCNNNN-Install-PowerShell.md

```diff
@@ -0,0 +1,176 @@
+
+This specification is an effort to help IT admins that wish to use new and updated releases of
+PowerShell 7. While this may not help customers that have post-deployment policy restrictions, it
+will help the majority of customers discover and update to the latest version of PowerShell for
+Windows.
+
+```
+As an admin,
+I can use Windows PowerShell 5.1 to provision multiple hosts with PowerShell 7 over PSRemoting,
+so that I can leverage new capabilities in PS7.
+```
+
+```
+As a user,
+I can discover and install PowerShell 7 from Windows PowerShell 5.1 or `cmd.exe` on my Windows machine,
+so that I can leverage new capabilities in PS7.
+```
+
+Additional scenarios are enabled when considering a native command versus a cmdlet. While Windows
+users would find benefit and familiarity to a cmdlet, a native command opens up scenarios where
+PowerShell is not pre-installed. Additionally, several installation issues due to open files are
+avoided by using a native command and closing any open sessions of PowerShell.
+
+## Shipping in Windows
+
+There are several issues that prevent shipping PowerShell 7 in Windows:
+
+- Size
+- Support lifecycle differences
+
+### Size constraints
+
+ A complete installation of PowerShell 7 currently is around 200 MB (unpacked). This is due to
+ including everything (modules and .NET assemblies) to provide parity with what Windows PowerShell
+ 5.1 with .NET Framework provides. There is currently no plan by Windows or the .NET team to ship
+ newer .NET in Windows, so PS7 will need to continue to ship .NET with PS7.
+
+### Support lifecycle
+
+ Windows has a 5+5 year support lifecycle. .NET supports up to 3 years for Long Term Support (LTS)
+ versions. So even if size was not an issue, we cannot check in a version of PS7 with a .NET that is
+ not supported while Windows itself is supported.
+
+### Solution
+
+To solve these issues, the proposal is to ship a new command in Windows that would:
+
+- Be locally discoverable
+- Perform a simple download and install of the latest LTS release of PowerShell
+- Include parameter options to select Stable, Preview, or LTS releases
+
+## Goals/Non-Goals
+
+Goals:
+
+- By default, download and install the latest LTS release to the users CurrentUser Scope
+  - CurrentUser - default installation scope location $env:LOCALAPPDATA\Microsoft\pwsh. If the path
+    exists, remove and replace.
+  - AllUsers - Administrative users may explicit select this scope. AllUsers requires administrative
+    elevation and performs a silent install using the MSI defaults.
+- Allow users to use remoting for install at scale with Invoke-Command
+- Allow advanced users to specify channel Stable/Preview/LTS
+
+Non-goals:
+
+- Removal of PowerShell (Remove-PowerShell7)
+- Updating of PowerShell (Update-PowerShell7)
+- User selected versions prior to current (No version parameter)
+- Disconnected scenarios - Users that are air-gapped may download the appropriate release from GitHub.
```

Comment:
Do we need to worry about regions like China, EU, etc.. that may not have access to a US based Az blob URL?

Reply:
We need an error message+links to docs page.

Reply:
Add progress bar (message)  for download and install portion